api/MediaDevices.json

```diff
@@ -401,6 +401,39 @@
             "deprecated": false
           }
         }
+      },
+      "setCaptureHandleConfig": {
+        "__compat": {
+          "spec_url": "https://w3c.github.io/mediacapture-handle/identity/#dom-mediadevices-setcapturehandleconfig",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": "mirror",
+            "edge": "mirror",
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": "mirror",
+            "ie": {
+              "version_added": false
+            },
+            "oculus": "mirror",
+            "opera": "mirror",
+            "opera_android": "mirror",
+            "safari": {
+              "version_added": false
+            },
+            "safari_ios": "mirror",
+            "samsunginternet_android": "mirror",
+            "webview_android": "mirror"
+          },
+          "status": {
+            "experimental": false,
```

Comment:
Can you compute the correct value for this in the script? I'm requesting changes on many PRs because new things are usually experimental per guideline.